pkg/aks/suite_test.go

```diff
@@ -0,0 +1,18 @@
+package aks
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"golang.org/x/net/context"
+)
+
+var (
+	ctx = context.Background()
+)
+
+func TestAPIs(t *testing.T) {
```

Comment:
Why is this test file here if it's this small?

Reply:
This PR introduces `ginkgo` as testing framework and it requires initialization https://onsi.github.io/ginkgo/#bootstrapping-a-suite, this code will bootstrap the test runner and start tests.